node/src/main.rs

```diff
@@ -149,14 +151,17 @@ async fn run(matches: &ArgMatches<'_>) -> Result<()> {
                 committee,
                 &store,
                 parameters,
-            );
+            )
         }
         _ => unreachable!(),
-    }
+    };
 
     // Analyze the consensus' output.
```

Comment:
If I get it correctly, this function returns when we run the dag without consensus, right?

Reply:
Correct, because its input channel is closed.